transports/transport-rest/transport-rest-vertx/src/main/java/io/servicecomb/transport/rest/vertx/VertxRestTransport.java

```diff
@@ -16,33 +16,57 @@
 
 package io.servicecomb.transport.rest.vertx;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
 import io.servicecomb.core.Const;
 import io.servicecomb.core.Invocation;
 import io.servicecomb.core.transport.AbstractTransport;
+import io.servicecomb.foundation.common.net.NetUtils;
 import io.servicecomb.foundation.common.net.URIEndpointObject;
 import io.servicecomb.foundation.vertx.SimpleJsonObject;
 import io.servicecomb.foundation.vertx.VertxUtils;
 import io.servicecomb.swagger.invocation.AsyncResponse;
 import io.servicecomb.transport.rest.client.RestTransportClient;
 import io.servicecomb.transport.rest.client.RestTransportClientManager;
 import io.vertx.core.DeploymentOptions;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.stereotype.Component;
 
 @Component
 public class VertxRestTransport extends AbstractTransport {
     private static final Logger log = LoggerFactory.getLogger(VertxRestTransport.class);
+
     @Override
     public String getName() {
         return Const.RESTFUL;
     }
 
     @Override
+    public int getOrder() {
+        return -1000;
+    }
+
+    @Override
+    public boolean canInit() {
+        setListenAddressWithoutSchema(TransportConfig.getAddress());
+
+        URIEndpointObject ep = (URIEndpointObject) getEndpoint().getAddress();
+        if (ep == null) {
```

Comment:
no listen address in microservice.yaml, that's mean will not register publish endpoint to SC.